package.json

```diff
@@ -21,5 +21,8 @@
   "homepage": "https://github.com/EBM2018/filrouge-boilerplate#readme",
   "devDependencies": {
     "concurrently": "^4.1.0"
+  },
+  "dependencies": {
+    "mongodb": "^3.1.13"
```

Comment:
Do we need this ?

Reply:
Don't think we do @NansD - feel free to remove it.

Reply:
Done actually.